opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeForm.tsx

```diff
@@ -138,11 +143,21 @@ CsvMapperRepresentationAttributeFormProps
         />
       </div>
       <div>
-        <CsvMapperRepresentationAttributeOptions
-          attribute={attribute}
-          indexRepresentation={indexRepresentation}
-        />
+        {
+          (attribute.type === 'date' || attribute.multiple)
```

Comment:
maybe put this test in a dedicated function `hasOptions` to encapsulate this logic

Reply:
I could but I decide to not do it because in the next chunk, I will remove this condition as the dialog should be available to each representation with on option of adding a default value.

Reply:
ok, clear !